db/migrate/20210919213448_drop_gyms.rb

```diff
@@ -0,0 +1,7 @@
+class DropGyms < ActiveRecord::Migration[5.2]
+  def change
+    remove_foreign_key :gym_members, :gyms
+    remove_foreign_key :events, :gyms
+    drop_table :gyms
+  end
+end
```

Comment:
Nice work using migrations vs. dropping db tables

Reply:
100% - hadn't seen this before as an option